src/base/bittorrent/session.cpp

```diff
@@ -417,6 +417,13 @@ Session::Session(QObject *parent)
     , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), MixedModeAlgorithm::TCP
         , clampValue(MixedModeAlgorithm::TCP, MixedModeAlgorithm::Proportional))
     , m_multiConnectionsPerIpEnabled(BITTORRENT_SESSION_KEY("MultiConnectionsPerIp"), false)
+    , m_validateHTTPSTrackers(BITTORRENT_SESSION_KEY("ValidateHTTPSTrackers"), false)
+    , m_sslVersion(BITTORRENT_SESSION_KEY("SSLVersion"), SSLVersion::TLS12
+#if (BOOST_VERSION >= 106900)
+        , clampValue(SSLVersion::TLS11, SSLVersion::TLS12, SSLVersion::TLS13))
```

Comment:
Oops, should this just be:
`, clampValue(SSLVersion::TLS11, SSLVersion::TLS13))`
and L425:
`, clampValue(SSLVersion::TLS11)`?

Reply:
> Oops, CI is failing here. Should this just be:
> `, clampValue(SSLVersion::TLS11, SSLVersion::TLS13))`

yes.

> and L425:
> `, clampValue(SSLVersion::TLS11)`?

The current L425 is correct.